.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,27 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 13.0.1
+    - name: Build with Maven & Quarkus
+      run: mvn test
+    - uses: redhat-cop/github-actions/s2i@v1
+      with:
+        base: registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift
```

Comment:
should we be using latest here or should be pinning to a version (1.8)?

Reply:
Good point - pinned it and I'll update `git-api` as well.